### DISCUSSION TITLE
Metadata Validation Errors blocking 'publish'

### DISCUSSION THREAD

**Discussion by njmei** [dandi/helpdesk#102] [Q&A] [answered]
Hi All,

I'm trying to determine why the metadata for some draft DANDI Sets are failing to validate and blocking the ability to `publish`:

https://dandiarchive.org/dandiset/000569
https://dandiarchive.org/dandiset/000570

![image](https://github.com/dandi/helpdesk/assets/43766899/f702df3e-b291-4c15-8ab2-26f3de916390)

It seems the validation errors I see are of these three flavors:
```
contributor: string does not match regex "^([\w\s\-\.']+),\s+([\w\s\-\.']+)$"
contributor: unexpected value; permitted: 'Organization'
contributor: string does not match regex "^https://ror.org/[a-z0-9]+$" 
```

What's weird is that the web UI at least seems to think the metadata I've used for contributors is valid?
![image](https://github.com/dandi/helpdesk/assets/43766899/1e02e6c2-bd78-4bb4-be60-3d05444337d1)

Any help would be greatly appreciated!

Best,

Nick

**Comment by satra** [comment, marked as answer]
@njmei - the names have to be of the form `last, first ...`

**Reply by njmei** [reply]
Many thanks @satra. That resolved the issue!

It would definitely be nice if the error messages on the website actually pointed out the `last, first` order issue explicitly rather than the red herring error messages that were displayed.

**Reply by satra** [reply]
completely agree - we are trying to incorporate a few additional checkers that should do so.
